### DISCUSSION TITLE
BUG of r.univar in Grass84 (Windows) with qgis 3.38.2 (no error with Grass83 and Qgis 3.38.0)

### DISCUSSION THREAD

**Discussion by Rivages** [OSGeo/grass#4283] [General]
Hi,

I use a lot of grass functions using R and system(cmd) with grass command.
I update my Qgis 3.38-2 with grass84.

An error occurs

With grass83 (i reload it now on https://wingrass.fsv.cvut.cz/grass83/), I have this result
ATTENTION: MASK already exists and will be overwritten
[1] "r.univar --quiet --overwrite map=MNTmask1a output=C:\\Cartino2D\\France/MAMP2024_GMSH/C00_026384km_X889200Y6237761/univar.txt"
C:\GRASS83\etc\grass83.py:298: SyntaxWarning: invalid escape sequence '\.'
  "%([#0 +-]*)([0-9]*)(\.[0-9]*)?([hlL]?[diouxXeEfFgGcrsa%])", fmt
ATTENTION: Le verrouillage concurrent de jeux de cartes n'est pas pris en
           charge dans Windows
Read 2 items

With Qgis3.38.2 and grass84, i have this error
[1] "r.univar --quiet --overwrite map=MNTmask1a output=C:\\Cartino2D\\France/MAMP2024_GMSH/C00_026384km_X889200Y6237761/univar.txt"
ATTENTION: Le verrouillage concurrent de jeux de cartes n'est pas pris en
           charge dans Windows
ATTENTION: Read error: We're not at the start of a row
ATTENTION: Erreur -72 de compression ZSTD : Src size is incorrect
ATTENTION: Erreur -72 de compression ZSTD : Src size is incorrect
ATTENTION: Erreur -72 de compression ZSTD : Src size is incorrect
ATTENTION: Read error: We're not at the start of a row
ATTENTION: Erreur -72 de compression ZSTD : Src size is incorrect
ERREUR : Erreur lors de la d�compression des donn�es raster fp pour la
         ligne 8 de <MNTmask1a> : code d'erreur -1
ATTENTION: Erreur de d�compression LZ4
Appuyez sur une touche pour continuer... 

Before, with With Qgis3.38.0 and grass83, i have no problems

I have any problems with r.in.gdal; v.in.ogr, r.mask, r.resamp.stats...

Best regards
Frédéric

**Comment by neteler** [comment]
> With grass83 (i reload it now on https://wingrass.fsv.cvut.cz/grass83/), I have this result
> ATTENTION: MASK already exists and will be overwritten
> [1] "r.univar --quiet --overwrite map=MNTmask1a output=C:\Cartino2D\France/MAMP2024_GMSH/C00_026384km_X889200Y6237761/univar.txt"
> C:\GRASS83\etc\grass83.py:298: SyntaxWarning: invalid escape sequence '.'
> "%([#0 +-]_)([0-9]_)(.[0-9]*)?([hlL]?[diouxXeEfFgGcrsa%])",

This is a Python 3.12 issue which has been fixed in GRASS GIS 8.4.0.

The other GRASS GIS 8.4 error `ATTENTION: Read error: We're not at the start of a row` I suggest to report as an issue:
https://github.com/OSGeo/grass/issues

**Reply by Rivages** [reply]
I have created an issue: https://github.com/OSGeo/grass/issues/4284
Thanks
